code/modules/events/disease_outbreak.dm

```diff
@@ -9,13 +9,28 @@
 		D = new virus_type()
 	else
 		var/datum/disease/advance/A = new /datum/disease/advance
+		// Try generating a random spreading virus. If that fails just use the last generated one
+		for(var/i in 1 to 100)
```

Comment:
I really don't like this. It seems like it just generates up to 100 random viruses at once until it finds one that's acceptable.
Instead of doing this, why not just say, make the 1st symptom sneezing, then auto-gen the rest?
That would mean you get a valid result in one generation, as opposed to taking up to 100.

Reply:
Adding sneezing as a fixed symptom would make it very meta-able. Now it can also spread without. And can for instance be touch spread.

Ive tested the call a few times. It made a functioning virus every time below the 20 tries on average.

Reply:
I'm going to leave an objection on this until your generation is re-written to avoid re-rolling, and instead use a system that only generates once, but generates in a way that is guaranteed to produce a correct output that one time.